### DISCUSSION TITLE
Creating Table Headers

### DISCUSSION THREAD

**Discussion by avalon60** [Akascape/CTkTable#10] [Q&A] [answered]
Hi. I am having problems trying to create column headers. I tried 2 approaches:

Approach 1:

```
        connection_table = ['Connection Name', 'Username', 'Connect String', 'Contactable', 'Launch']
        self.tbv_connections = CTkTable(master=frm_cscan_widgets, values=connection_table)
        self.tbv_connections.grid(row=0, column=0)
```
This resulted in:
![image](https://github.com/Akascape/CTkTable/assets/89534395/670ee948-1a70-49c9-8bba-db79a1d70c14)

Approach 2:
```
        connection_table = ['Connection Name', 'Username', 'Connect String', 'Contactable', 'Launch']
        self.tbv_connections = CTkTable(master=frm_cscan_widgets)
        self.tbv_connections.add_row(values=connection_table)
```

This resulted in:

![image](https://github.com/Akascape/CTkTable/assets/89534395/8292db10-d533-41ea-9220-75cc2873bb5e)

Can you please advise how I do this?

Thanks.

**Comment by Akascape** [comment]
You mean you want a vertical header? I will have to add it.
Horizontal top header can be achieved using `header_color=color_name`

**Reply by avalon60** [reply]
No I want the headers to look normal - like this:

![image](https://github.com/Akascape/CTkTable/assets/89534395/bd8b8ae8-ac36-4d1f-a560-a9b3303ab6ca)

The vertical stack in the first image, is how it's coming out in the table for some reason. I want the labels to occupy individual columns. So for example 'Connection Name', would be in the first button (0, 0) of the table.

At least that's how they look now - I want to improve the above display using CTkTable.
NOTE: The labels in the headings above (blue image) are slightly different, but hopefully you get the idea.

**Reply by avalon60** [reply, marked as answer]
Problem solved. I had missed the outer brackets:
`
connection_table = [['Connection Name', 'Username', 'Connect String', 'Contactable', 'Launch']]`
